lib/pages/not_found.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:tfc_hmi/widgets/base_scaffold.dart';
+import 'package:beamer/beamer.dart';
+
+class PageNotFound extends StatelessWidget {
+  const PageNotFound({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    // TODO: Maybe show the path used
+    var actions = <Widget>[
+      TextButton(
+          onPressed: () => context.beamToNamed('/'),
```

Comment:
this could brake any time where / is not a page, like if we would make /home

not sure what the correct way is here